diff --git components/libsodium/libsodium/src/libsodium/include/sodium/crypto_generichash_blake2b.h src/libsodium/include/sodium/crypto_generichash_blake2b.h
index 7b0c0820..5443c483 100644
--- components/libsodium/libsodium/src/libsodium/include/sodium/crypto_generichash_blake2b.h
+++ components/libsodium/libsodium/src/libsodium/include/sodium/crypto_generichash_blake2b.h
@@ -20,7 +20,7 @@ extern "C" {
 # pragma pack(push, 1)
 #endif

-typedef CRYPTO_ALIGN(64) struct crypto_generichash_blake2b_state {
+typedef struct CRYPTO_ALIGN(64) crypto_generichash_blake2b_state {
     uint64_t h[8];
     uint64_t t[2];
     uint64_t f[2];
